front/app/components/EventCard/EventInformation/index.tsx

```diff
@@ -194,8 +198,14 @@ const EventInformation = ({ event, titleFontSize }: Props) => {
           ml="auto"
           width={'100%'}
           bgColor={theme.colors.tenantPrimary}
+          tabIndex={0}
           onClick={() => {
-            clHistory.push(`/events/${event.id}`, { scrollToTop: true });
+            goToEvent();
+          }}
+          onKeyDown={(e) => {
+            if (e.key === 'Enter') {
+              goToEvent();
+            }
           }}
```

Comment:
I've tested and this code doesn't look necessary. It works fine with the keyboard by just having `onClick={goToEvent}`.

(It would have been strange that we needed these changes. Hitting enter works in other places where we use this button. And also, a button is reachable by tab by default, so it shouldn't need a tabIndex={0}. 

Also, if there was a real problem, we would have to ideally fix it at the Button component level, but not on a per-case basis so we don't keep the same problem in other places.)